api/v1/db/oracledb/player-dao.js

```diff
@@ -74,6 +74,59 @@ const getPlayerByGameIdAndPlayerId = async (id, pid, isPost = false) => {
   }
 };
 
+const cleanPlayerCardsByPlayerId = async (playerId, connection) => {
+  const sqlParams = [playerId];
+  const deleteSqlQuery = `
+  DELETE FROM PLAYER_CARDS WHERE PLAYER_ID = :playerId
+  `;
+  const result = await connection.execute(deleteSqlQuery, sqlParams);
+  return result;
+};
+
+const deletePlayerByPlayerId = async (playerId, passedConnection) => {
+  const isPassedConnection = Boolean(passedConnection);
+  const connection = isPassedConnection ? passedConnection : await conn.getConnection();
+  try {
+    await cleanPlayerCardsByPlayerId(playerId, connection);
+    const sqlParams = [playerId];
+    const deleteSqlQuery = `
+    DELETE FROM PLAYERS WHERE PLAYER_ID = :playerId
+    `;
+    const result = await connection.execute(deleteSqlQuery, sqlParams,
+      { autoCommit: !isPassedConnection });
+    return result;
+  } finally {
+    if (!isPassedConnection) {
+      connection.close();
+    }
+  }
+};
+
+
+const deletePlayersByGameId = async (gameId) => {
```

Comment:
Since this function is called by `deleteGameByGameId` in the game dao, could you modify this function so that it won't commit any changes?